docs/make.jl

```diff
@@ -5,6 +5,8 @@ using Documenter
 using DocumenterCitations
 using Gabs
 using QuantumInterface
+import Latexify
```

Comment:
```suggestion
import Latexify: latexify
```
Does this work here? Might be better since we only need to import one function from Latexify into the docs make file.